sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/_tests_/e2e/application-restriction-bypass.aest.controller.bypassRestriction.e2e-spec.ts

```diff
@@ -0,0 +1,279 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  RestrictionCode,
+  createE2EDataSources,
+  createFakeUser,
+  saveFakeApplication,
+  saveFakeApplicationRestrictionBypass,
+  saveFakeStudentRestriction,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import {
+  ApplicationStatus,
+  NoteType,
+  OfferingIntensity,
+  Restriction,
+  RestrictionActionType,
+  RestrictionBypassBehaviors,
+  User,
+} from "@sims/sims-db";
+import {
+  ACTIVE_BYPASS_FOR_STUDENT_RESTRICTION_ALREADY_EXISTS,
+  APPLICATION_IN_INVALID_STATE_FOR_APPLICATION_RESTRICTION_BYPASS_CREATION,
+  STUDENT_RESTRICTION_IS_NOT_ACTIVE,
+} from "../../../../constants";
+
+describe("ApplicationRestrictionBypassAESTController(e2e)-bypassRestriction", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let sharedMinistryUser: User;
+  let ptssrRestriction: Restriction;
+  let endpoint: string;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sharedMinistryUser = await db.user.save(createFakeUser());
+    ptssrRestriction = await db.restriction.findOneBy({
+      restrictionCode: RestrictionCode.PTSSR,
+    });
+    endpoint = "/aest/application-restriction-bypass";
+  });
+
+  it("Should be able to create a bypass when there is not an active bypass for the same student.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource, undefined, {
+      offeringIntensity: OfferingIntensity.partTime,
+    });
+
+    const studentRestriction = await saveFakeStudentRestriction(db.dataSource, {
+      student: application.student,
+      restriction: ptssrRestriction,
+      application,
```

Comment:
As a suggestion, the application can be removed to ensure that bypass can be applied without the relationship in place.